builtin-func.l

```diff
@@ -314,9 +322,9 @@ void finish_alternative_mode()
 // FreeBSD doesn't support LeakSanitizer
 #if defined(USING_ASAN) && !defined(__FreeBSD__)
 	#include <sanitizer/lsan_interface.h>
-	#define BIFCL_LSAN_DISABLE(x) __lsan_disable(x)
+	#define BIFCL_LSAN_DISABLE(...) __lsan_disable()
```

Comment:
For those specifically, do you know why there's actually an `x` ?  Could you let go of the `...` Actually passing something for `x` would've actually failed I presume as they are declared with `(void)`

https://github.com/llvm/llvm-project/blob/d515ecca683422fbb46e23d30f0016efcbc84f01/compiler-rt/include/sanitizer/lsan_interface.h#L23-L24

Reply:
I swear there was a technical reason why we needed the argument there in the first place, but CI seems to work fine without it. I removed it here and in binpac.